advocate_europe/assets/scss/components/_comments.scss

```diff
@@ -0,0 +1,172 @@
+.black-divider {
+    display: none;
+}
+
+.comment-box {
+    & textarea {
+        margin-top: rem(10px);
+        border-radius: 5px;
+        background-color: $ae-lighter-gray;
+        border: 1px solid $ae-light-gray-shadow;
+        @include rem(font-size, 14px);
+    }
+
+    .submit-button {
+        @extend .btn;
```

Comment:
Great, thanks!
The margin shouldn't have been there. I applied it to the whole ideadetail-comments class now.